.github/workflows/review.yaml

```diff
@@ -39,7 +39,7 @@ jobs:
             requirements-dev.txt
 
       - name: Install dependencies
-        run:  pip install -r requirements-dev.txt
+        run: ./tools/install-deps
```

Comment:
Note - this is not currently working on windows - see pull request https://github.com/mewler/mewbot/pull/49 - in particular the additions to tools/path.sh.

As these are windows only fixes, seemed easier to include them in the general windows docs. But I can pull them out and make em a seperate request if you'd prefer for neatness

Reply:
I was wondering about adding direct invoke to the docs - if we're committed to having the shell scripts just be a tight wrapper around the underlying python invoke (which makes sense to me) - I'll go ahead and add it in as an alternative to installing a posix shell (which is a bit of a faff) to https://github.com/mewler/mewbot/pull/49